Applications/Embedding/README.md

```diff
@@ -0,0 +1,39 @@
+# Embedding
+
+This application contains embedding layer example with two example model
+
+## Example Model Structure
+
+There are two structure ready for the example
+
+1. Linear Embedding structure
+ (Input:           10:1:1:4 )
+ (Embedding:       10:1:4:8 )
+ (Flatten:         10:1:1:32)
+ (FullyConnected:  10:1:1:1 )
+
+2. Splitted data and seperate embedding structure
+ (split:           10:1:2:2 ) # splited into 10:1:1:1, 10:1:1:1)
```

Comment:
it might be good to add a `input  : 10:1:2:2` as well.

Reply:
Updated,